development-docs/DEV_GUIDE.md

```diff
@@ -19,209 +25,286 @@ This document gives a detailed breakdown of the various build processes and opti
             -e "s#(image|value): quay.io/strimzi/([a-zA-Z0-9-]+:[0-9.]+)#\1: $DOCKER_REGISTRY/$DOCKER_ORG/\2#" \
             -e "s#([0-9.]+)=quay.io/strimzi/([a-zA-Z0-9-]+:[a-zA-Z0-9.-]+-kafka-[0-9.]+)#\1=$DOCKER_REGISTRY/$DOCKER_ORG/\2#" \
-            install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml
     ```
 
-   This updates `060-Deployment-strimzi-cluster-operator.yaml`, replacing all the image references (in `image` and `value` properties) with ones with the same name but with the repository changed.
-
-5. Then you can deploy the Cluster Operator by running: 
-
-        kubectl create -f install/cluster-operator
-
-6. Finally, you can deploy the cluster custom resource running:
-   
-        kubectl create -f examples/kafka/kafka-ephemeral.yaml 
+   **OS X**
+    ```
+    sed -E -i '' -e "s#(image|value): quay.io/strimzi/([a-z0-9-]+):latest#\1: $DOCKER_REGISTRY/$DOCKER_ORG/\2:latest#" \
+                 -e "s#(image|value): quay.io/strimzi/([a-zA-Z0-9-]+:[0-9.]+)#\1: $DOCKER_REGISTRY/$DOCKER_ORG/\2#" \
+                 -e "s#([0-9.]+)=quay.io/strimzi/([a-zA-Z0-9-]+:[a-zA-Z0-9.-]+-kafka-[0-9.]+)#\1=$DOCKER_REGISTRY/$DOCKER_ORG/\2#" \
+                  packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml
+    ```
 
-## Build Pre-Requisites
+   This updates `060-Deployment-strimzi-cluster-operator.yaml`, replacing all the image references (in `image`
+   and `value` properties) with ones with the same name but with the repository changed.
+   > *Note*: please ensure you don't commit these changes accidentally.
 
-### Command line tools
 
-To build this project you must first install several command line utilities and a Kubernetes or OpenShift cluster.
+5. The installation files assume you're installing into the namespace `myproject`. If you wish to use a different one,
+   you'll need to replace it in the installation files.
+   \
+   \
+   **Linux**
 
-- [`make`](https://www.gnu.org/software/make/) - Make build system
-- [`mvn`](https://maven.apache.org/index.html) (version 3.5 and above) - Maven CLI
-- [`helm`](https://helm.sh/) (version 3) - Helm Package Management System for Kubernetes 
-- [`asciidoctor` and `asciidoctor-pdf`](https://asciidoctor.org/) - Documentation generation. 
-    - Use `gem` to install latest version for your platform.
-- [`yq`](https://github.com/mikefarah/yq) - (version 4.2.1 and above) YAML manipulation tool. 
-    - **Warning:** There are several different `yq` YAML projects in the wild. Use [this one](https://github.com/mikefarah/yq). You need version 4 or above.
-- [`docker`](https://docs.docker.com/install/) - Docker command line client
-- [`shellcheck`](https://github.com/koalaman/shellcheck) - ShellCheck is a GPLv3 tool that gives warnings and suggestions for bash/sh shell scripts.
+       sed -Ei "s/myproject/<desired_namespace>/g" packaging/install/cluster-operator/*.yaml
```

Comment:
This should be consistent with the OS X example - using `*RoleBinding*.yaml` instead of `*.yaml`